### PR TITLE
Add some more redirects for additional identity-* repos

### PR DESCRIPTION
**Pull request by zachmargolis**
I added some explicit tests because I think these are non-standard cases

### CODE REVIEW

**Comment by wslack** [comment]
@zachmargolis where those other pages.18f.gov site links ever live?

**Comment by zachmargolis** [comment]
@wslack yup! they are live now and we pushed changes with `<meta>` redirects to where we wanted them to go

**Comment by zachmargolis** [comment]
@brandocalrissian ok! I did a teeny amount of custom development to add a new feature, finally got the tests to pass, please let me know if this is acceptable? Thanks!

File: test/unit/test_lib.js [hunk 1]
```diff
@@ -26,10 +26,10 @@ test('lib.getPageConfigs', (t) => {
...
   t.ok(configs);
   t.equal(configs.length, 4);
-  t.same(configs[0], { to: 'test1', from: 'test1', toDomain: '18f.gov' });
-  t.same(configs[1], { to: 'test2', from: 'test2', toDomain: '18f.gov' });
-  t.same(configs[2], { to: 'test3to', from: 'test3from', toDomain: '18f.gov' });
-  t.same(configs[3], { to: 'test4to', from: 'test4from', toDomain: 'anotherdomain.gov' });
+  t.same(configs[0], { to: 'test1', from: 'test1', toDomain: '18f.gov', toPath: '' });
+  t.same(configs[1], { to: 'test2', from: 'test2', toDomain: '18f.gov', toPath: '' });
+  t.same(configs[2], { to: 'test3to', from: 'test3from', toDomain: '18f.gov', toPath: '' });
+  t.same(configs[3], { to: 'test4to', from: 'test4from', toDomain: 'anotherdomain.gov', toPath: '' });
```
Comment (brandocalrissian) [inline test/unit/test_lib.js]: Can we adjust the test case so at least one of these toPath values is non-empty to verify we're doing the right thing?

Comment (zachmargolis) [inline test/unit/test_lib.js]: Sure! PTAL @ f81ad783199e58cd4fe2ef211b00f909e2b10ec4

**Pull request merged**